source/constructing-lattices.md

```diff
@@ -221,27 +222,27 @@ of the preceding `item` as explained in the [Branch Coordinates Construction](#s
 To adjust the longitudinal placement of an `item`, 
 the `placement` component of an `item` can be used.
 When there is a `placement` component, figure {numref}`f:superposition` shows how the line `item` 
-is positioned with respect to a reference line `item`. 
-If the `reference_item` is not specified, or set to `BEGINNING`, the beginning of the `line` is used. 
+is positioned with respect to a `"base"` line `item`. 
 
 The components of `placement` are:
 ```{code} yaml
-offset            # Optional Real [m]. Longitudinal offset of the line item. Default is zero.
-origin            # Optional switch. Line item origin point. Default is ENTRANCE_END.
-reference_item    # Optional string. Reference line item. Default is a blank string which 
-                  #  indicates the previous element or the beginning of the `line` if there
-                  #  is no previous element.
-reference_origin  # Optional switch. Reference line item origin point. Default is EXIT_END.
+offset       # Optional Real [m]. Longitudinal offset of the line item. Default is zero.
+to_point     # Optional switch. Line `item` offset end point. Default is ENTRANCE_END.
+base_item    # Optional string. Reference line item. Default is a blank string which 
```

Comment:
Change "Reference line item" to "Base line item" above?
Does "Default is a blank string which indicates the previous element or the beginning of the `line` if there is no previous element." mean that 'base_item="" ' indicates that the base_item is the previous element if any or the beginning of the line otherwise? If so, it may need to be clearer especially with "If the `base_item` is not specified, or set to `BEGINNING`, the beginning of the `line` is used. " below.

Reply:
I have revised this. Please take a look.